certbot/certbot/_internal/client.py

```diff
@@ -548,14 +547,6 @@ def enhance_config(self, domains, chain_path, ask_redirect=True):
         for config_name, enhancement_name, option in enhancement_info:
             config_value = getattr(self.config, config_name)
             if enhancement_name in supported:
-                if ask_redirect:
-                    if config_name == "redirect" and config_value is None:
-                        config_value = enhancements.ask(enhancement_name)
```

Comment:
This change also has the effect of no longer setting up a redirect unless the user manually requested it with `--redirect`.

To fix this, I'd recommend keeping code like this here:
```
if config_name == "redirect" and config_value is None:
    config_value = True
```